code/WorkInProgress/Cael_Aislinn/Rust/fuel_control.dm

```diff
@@ -74,24 +74,20 @@
 		[announce_stageprogression == 2 ? 	"Broadcasting"	: "<a href='?src=\ref[src];announce_stageprogression=2'>\[Broadcast\]</a>"]<br>"}*/
 	dat += {"
 		<hr>
-		<b>Detected devices</b> <a href='?src=\ref[src];scan=1'>\[Refresh list\]</a>
+		<b>Detected devices</b>
 		<table border=1 width='100%'>
 		<tr>
 		<td><b>ID</b></td>
 		<td><b>Assembly</b></td>
 		<td><b>Consumption</b></td>
 		<td><b>Depletion</b></td>
-		<td><b>Duration</b></td>
-		<td><b>Next stage</b></td>
-		<td></td>
-		<td></td>
 		</tr>"}
 	for(var/obj/machinery/power/rust_fuel_injector/I in connected_injectors)
 
 		dat += {"<tr>
 			<td>[I.id_tag]</td>"}
 		if(I.cur_assembly)
-			dat += "<td><a href='?src=\ref[I];toggle_injecting=1;update_extern=\ref[src]'>\[[I.injecting ? "Halt injecting" : "Begin injecting"]\]</a></td>"
+			dat += "<td><a href='?src=\ref[src];toggle_injecting=1;update_extern=\ref[src]'>\[[I.injecting ? "Halt injecting" : "Begin injecting"]\]</a></td>"
```

Comment:
Remove the update extern part of the href please.

Also set the association value of `toggle_injecting` to be `\ref[I]`